### PR TITLE
add new timers for "map" process and time for blocks to start working

### PR DESCRIPTION
**Pull request by VOD555**
Changes made in this Pull Request:
 - add timer for "map" process
 - add timer for 'waiting' time (the time needed for blocks to start working)

PR Checklist
------------
 - [x] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: pmda/parallel.py [hunk 1]
```diff
@@ -35,14 +35,17 @@ class Timing(object):
     store various timeing results of obtained during a parallel analysis run
     """
 
-    def __init__(self, io, compute, total, universe, prepare, conclude):
+    def __init__(self, io, compute, total, universe, prepare,
+                 conclude, distr, wait):
```
Comment (orbeckst) [inline pmda/parallel.py]: write out `distr` as `distribute` to make it clearer

File: pmda/parallel.py [hunk 2]
```diff
@@ -35,14 +35,17 @@ class Timing(object):
...
         self._io = io
         self._compute = compute
         self._total = total
         self._cumulate = np.sum(io) + np.sum(compute)
         self._universe = universe
         self._prepare = prepare
         self._conclude = conclude
+        self._distr = distr
```
Comment (orbeckst) [inline pmda/parallel.py]: write out `distr` – not sure what it means

File: pmda/parallel.py [hunk 3]
```diff
@@ -347,32 +360,39 @@ def run(self,
...
+                            self._dask_helper, pure=False)(
+                                bslice,
+                                self._indices,
+                                self._top,
+                                self._traj, )
+                        blocks.append(task)
+                    blocks = delayed(blocks)
+                time_distr = distr.elapsed
+
+                blocks_start = time.time()
```
Comment (orbeckst) [inline pmda/parallel.py]: Where is `blocks_start` used?

Comment (orbeckst) [reply pmda/parallel.py]: Add a comment to say what it measures.

**Comment by orbeckst** [comment]
Also CHANGELOG & docs!

**Comment by VOD555** [comment]
@orbeckst All the changes made in the PR have passed the tests. Please check whether there is something need to be modified.

**Comment by orbeckst** [comment]
Please fix PEP8 ...

**Comment by orbeckst** [comment]
Also issues in LeafLet that are related to the changes.

**Comment by orbeckst** [comment]
I fixed #83 and merged master into this branch so now all tests should pass.

**Pull request merged**